mash_scripts/pulpcore/3.4/pulpcore-3.4-el7.mash

```diff
@@ -0,0 +1,12 @@
+[pulpcore-3.4-el7]
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = pulpcore-3.4-el7
+inherit = True
+strict_keys = False
```

Comment:
Implies we are not signing them?

Reply:
At the moment correct, I didn't create a key etc yet